bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/ElectionRecord/ElectionRecordData.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using ElectionGuard.Decryption.Tally;
+using ElectionGuard.ElectionSetup;
+using ElectionGuard.Extensions;
+using ElectionGuard.Guardians;
+
+namespace ElectionGuard.Decryption.ElectionRecord;
+
+public record ElectionRecordData : DisposableRecordBase
```

Comment:
this file now holds al of the objects that must be serialized out (or in) as part of the election record

Reply:
This might cause an issue for a larger election such as 1M ballots.